executor/cook/config.py

```diff
@@ -49,12 +49,28 @@ def initialize_config(environment):
     """Initializes the config using the environment.
     Populates the default values for missing environment variables.
     """
+    executor_id = environment.get('MESOS_EXECUTOR_ID', 'executor')
+    sandbox_directory = environment.get('MESOS_SANDBOX', '')
+    default_progress_output_name = '{}.progress'.format(executor_id)
+    if sandbox_directory:
+        default_progress_output_file = '{}/{}'.format(sandbox_directory, default_progress_output_name)
+    else:
+        default_progress_output_file = default_progress_output_name
+
+    if 'EXECUTOR_PROGRESS_OUTPUT_FILE_ENV' in environment:
+        progress_output_file_env = environment.get('EXECUTOR_PROGRESS_OUTPUT_FILE_ENV')
+        logging.info('Progress location environment variable is {}'.format(progress_output_file_env))
```

Comment:
We can remove the duplication and move this after the `if`

Reply:
Changed. I previously wanted to output additional log only when `EXECUTOR_PROGRESS_OUTPUT_FILE_ENV` is specified.